spec/lib/services/account-api-service-spec.js

```diff
@@ -57,98 +70,193 @@ describe("Account API Service", function() {
         ]);
     });
 
-    after(function() {
+    after(function () {
         self.sandbox.restore();
     });
 
-    afterEach(function() {
+    afterEach(function () {
         self.sandbox.reset();
     });
 
-    it('should list users', function() {
-        waterline.localusers.find.resolves([user1, user2]);
-        return accountService.listUsers()
-        .then(function(users) {
-            expect(waterline.localusers.find).to.have.been.called;
-            expect(users.length).to.equal(2);
-            return Promise.map(users, function(user) {
-                expect(user.username).to.be.a('string');
+    describe('Users services', function () {
+
+        it('should list users', function () {
+            waterline.localusers.find.resolves([user1, user2]);
+            return accountService.listUsers()
+            .then(function (users) {
+                expect(waterline.localusers.find).to.have.been.called;
+                expect(users.length).to.equal(2);
+                return Promise.map(users, function (user) {
+                    expect(user.username).to.be.a('string');
+                    expect(user.password).to.be.an('undefined');
+                    expect(user.role).to.be.a('string');
+                    expect(user.privileges).to.be.instanceof(Array);
```

Comment:
For those user's property, we'd better check their content rather than only the type.